browser/src/map/handler/Map.Keyboard.js

```diff
@@ -140,9 +140,110 @@ L.Map.Keyboard = L.Handler.extend({
 	},
 
 	keyCodes: {
+
+		BACKSPACE:8,
+		TAB:      9,
+		enter:   13, 
+		SPACE :   32,
 		pageUp:   33,
 		pageDown: 34,
-		enter:    13
```

Comment:
Can we move enter after `pageDown:` line as it was before so it will not show modification?
Later it's easier to find last commit using git grep tool.

Reply:
(We will fix order and name later in separate PR, just want to keep access to the commit introducing that variable :) )

Reply:
Done